recipe/meta.yaml

```diff
@@ -192,18 +195,31 @@ outputs:
       run:
         - openblas   # [blas_impl == "openblas"]
         - mkl-devel  # [blas_impl == "mkl"]
-        - {{ pin_subpackage("liblapack", exact=True) }}      # [blas_impl != 'blis']
-        - {{ pin_subpackage("liblapacke", exact=True) }}     # [blas_impl != 'blis']
+        - {{ pin_subpackage("libblas", exact=True) }}        # [blas_impl != 'netlib']
+        - {{ pin_subpackage("libcblas", exact=True) }}       # [blas_impl != 'netlib']
+        - {{ pin_subpackage("liblapack", exact=True) }}      # [blas_impl not in ('blis', 'netlib')]
+        - {{ pin_subpackage("liblapacke", exact=True) }}     # [blas_impl not in ('blis', 'netlib')]
         - liblapack  {{ version }} *netlib                   # [blas_impl == 'blis']
         - liblapacke {{ version }} *netlib                   # [blas_impl == 'blis']
-        - {{ pin_subpackage("libcblas", exact=True) }}
-        - {{ pin_subpackage("libblas", exact=True) }}
+        # netlib builds are build in lapack-feedstock
+        - pin_run_as_build('libblas', exact=True)            # [blas_impl == 'netlib']
+        - pin_run_as_build('libcblas', exact=True)           # [blas_impl == 'netlib']
+        - pin_run_as_build('liblapack', exact=True)          # [blas_impl == 'netlib']
+        - pin_run_as_build('liblapacke', exact=True)         # [blas_impl == 'netlib']
```

Comment:
This is following @minrk's [comment](https://github.com/conda-forge/blas-feedstock/issues/66#issuecomment-800912104) - I've never used this function and don't know how it works, thought the `exact=True` makes no sense to me tbh...